src/main/java/net/fabricmc/fabric/mixin/resources/MixinMinecraftGame.java

```diff
@@ -16,64 +16,54 @@
 
 package net.fabricmc.fabric.mixin.resources;
 
-import com.google.common.base.Joiner;
 import com.google.common.collect.Lists;
 import net.fabricmc.fabric.impl.resources.ModResourcePackUtil;
 import net.minecraft.client.MinecraftClient;
 import net.minecraft.client.resource.DefaultClientResourcePack;
-import net.minecraft.resource.DefaultResourcePack;
-import net.minecraft.resource.ReloadableResourceManager;
 import net.minecraft.resource.ResourcePack;
 import net.minecraft.resource.ResourceType;
 import org.spongepowered.asm.mixin.Mixin;
-import org.spongepowered.asm.mixin.Shadow;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Inject;
-import org.spongepowered.asm.mixin.injection.ModifyVariable;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
 import org.spongepowered.asm.mixin.injection.callback.LocalCapture;
 
-import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.CompletableFuture;
 
 @Mixin(MinecraftClient.class)
 public class MixinMinecraftGame {
```

Comment:
These changes seem a bit out of scope for the current PR. While I'm not opposed to them, I think it'd better to move them out for the time being.